zenoh/__init__.pyi

```diff
@@ -349,6 +352,7 @@ class KeyExpr:
 _IntoKeyExpr = KeyExpr | str
 
 @final
+@_unstable
```

Comment:
I would rather put `@_unstable` above the `@final`, but it's a matter of taste.

Reply:
fixed